### PR TITLE
Adds the Captain's Wardrobe, removes a TON of Captain's Locker bloat items.

### PR DESCRIPTION
**Pull request by ShadeAware**
## About The Pull Request
Adds in a new Wardrobe, the Captain's Wardrobe, which holds all cosmetic Captain clothing. Clothing has been moved from the Locker to the vender because of this. 

First time editing a map so expect everything to go wrong. 🙃 

## Why It's Good For The Game
The Captain's locker is bloated to shit. This helps alleviate the problem and still let people have their fancy mix-and-match outfits.

## Changelog
:cl:
Add: The Captain's Wardrobe, a vender that holds all cosmetic Captain's clothing.
Remove: Most cosmetic items from the Captain's locker.
/:cl:

### CODE REVIEW

**Comment by ShadeAware** [comment]
ah cool, conflicts

**Comment by ghost** [comment]
F

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -1,39 +1,22 @@
-/obj/structure/closet/secure_closet/captains
+/obj/structure/closet/secure_closet/captains //Unbloated by ShadeAware aka Michael Labs
```
Comment (ghost) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: recommend changing comment to tell people that the other stuff is in vending/wardrobes

**Comment by ShadeAware** [comment]
Its getting there.

File: code/modules/vending/wardrobes.dm [hunk 2]
```diff
@@ -400,3 +400,31 @@
...
+					/obj/item/clothing/shoes/sneakers/brown = 2,
+					/obj/item/clothing/under/rank/captain = 1,
+					/obj/item/clothing/under/rank/captain/skirt = 1,
+					/obj/item/clothing/head/caphat = 1,
+					/obj/item/clothing/head/caphat/parade = 1,
+					/obj/item/clothing/head/caphat/beret = 1,
+					/obj/item/clothing/head/caphat/beret/white = 1,
+					/obj/item/clothing/under/captainparade = 1,
+					/obj/item/clothing/suit/captunic = 1,
+					/obj/item/clothing/under/rank/captain/femformal = 1, //Fuck your Citadel Edits and fuck locker bloat, we VENDERSTATION now.
```
Comment (Ghommie) [inline code/modules/vending/wardrobes.dm]: Remove comment.

**Review by Ghommie — CHANGES_REQUESTED** [review]
Also merge conflicts

**Comment by ShadeAware** [comment]
Im just going to try to redo this entirely.

**Pull request closed without merging**